From 88fdba3b148df918ad24bd2d5bb48c302651acfc Mon Sep 17 00:00:00 2001
From: Kris Bahnsen <kris@embeddedTS.com>
Date: Tue, 21 Jun 2022 22:23:33 +0000
Subject: [PATCH 2/4] sound: Add config option for SOC_IMX_MAX98357A

---
 sound/soc/fsl/Kconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 8e05d092790e..be688ff4eaa2 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -268,6 +268,16 @@ config SND_SOC_EUKREA_TLV320
 	  Enable I2S based access to the TLV320AIC23B codec attached
 	  to the SSI interface
 
+config SND_SOC_IMX_MAX98357A
+	tristate "SoC Audio support for i.MX boards with MAX98357A"
+	depends on OF
+	select SND_SOC_MAX98357A
+	select SND_SOC_IMX_PCM_DMA
+	select SND_SOC_IMX_AUDMUX
+	select SND_SOC_FSL_SSI
+	help
+	  Support for MAX98357A on TS-Terumo board
+
 config SND_SOC_IMX_ES8328
 	tristate "SoC Audio support for i.MX boards with the ES8328 codec"
 	depends on OF && (I2C || SPI)
-- 
2.25.1

